docs/developer-docs/getting-started/overview-of-icp.mdx

```diff
@@ -244,7 +245,7 @@ Currently, subnets contain between 13 and 40 nodes, and most of them are geograp
 </tr>
 
 <tr>
-  <td> Upgradability </td>
+  <td> Upgradeability </td>
```

Comment:
I believe that it should be Upgradability because we use "Upgradable" in some places: https://github.com/dfinity/portal/pull/2517/files